src/components/EpisodeCard.tsx

```diff
@@ -0,0 +1,38 @@
+import IEpisode from "../interfaces/IEpisode";
+import cleanString from "../util/cleanString";
+import padNumber from "../util/padNumber";
+import "./episodeCard.css";
+
+interface EpisodeCardProps {
+    key: number;
+    episode: IEpisode;
+}
+
+export function EpisodeCard(props: EpisodeCardProps): JSX.Element {
```

Comment:
Since you don't need the key, you could choose to destruct the props here to be {episode}. This avoids using props[key] later on in the component.